lib/recaptcha.js

```diff
@@ -3,7 +3,8 @@ module.exports = function(self) {
     async getRecaptchaSecret (req, self) {
       const globalDoc = await self.apos.global.find(req, {}).toObject();
 
-      return self.options.recaptchaSecret || globalDoc.recaptchaSecret;
+      return self.options.recaptchaSecret ? self.options.recaptchaSecret
+        : globalDoc.useRecaptcha ? globalDoc.recaptchaSecret : null;
```

Comment:
There are plans for a "monitor" functionality in A3 that would handle things like turning globl doc properties into apos object options automatically, replacing the old apostrophe-override-options module with something similar to how assembly site piece properties are able to set startup-time options in the apos object. I think that would be better than hacking it in for individual modules, so I'd like to kick this can down the road for now.